finagle/h2/src/main/scala/io/netty/handler/codec/http2/H2FrameCodec.scala

```diff
@@ -210,7 +210,6 @@ object H2FrameCodec {
       val fr = new Http2InboundFrameLogger(new DefaultHttp2FrameReader, frameLogger)
       new DefaultHttp2ConnectionDecoder(conn, encoder, fr)
     }
-    decoder.localSettings(settings)
```

Comment:
Well, I *think* it prevents any of the configured http2 settings from being sent during connection establishment.  We're going against the grain in netty (as you'll see at the top of this file}, so we probably need to do some poking to figure out if this feature has been removed from netty entiirely.  I don't think it's a hard blocker, but we may need to remove a bunch of the settings from the config.